projects/cashmere/src/lib/sass/tables.scss

```diff
@@ -26,10 +26,10 @@ $row-selected-font-color: $white;
 $row-selected-border-color: $white;
 
```

Comment:
Add variable for transparent border
``` $table-border-transparent: 1px solid transparent;```

Reply:
Thanks @jacquie - makes sense and made the change.  This one is actually an annoying bit of styling that I had to add.  What I was finding was that when I added the 2px border to `<tbody>` on a table with `width: 100%`, the table was overflowing it's container by the width of the border (so horizontal scroll bars were appearing).  By adding this transparent border to the overall table, it seemed to inset the table to allow the tbody borders to appear without extending past 100%.  @corykon - not sure if there's something I could have done differently to avoid this, but this was the only solution I could come up with that worked.